components/docs-chef-io/README.md

```diff
@@ -50,13 +50,15 @@ is updated in a repository, those changes won't appear in https://docs.chef.io u
 Hugo modules are updated to the new commit and documentation files are updated
 in the `_vendor` directory.
 
-This can be useful if you want to update documentation in `chef/automate`, but
-don't want it to appear on https://docs.chef.io until a later date.
+We use Expeditor to submit a pull request to `chef/chef-web-docs` when Automate is promoted to stable.
```

Comment:
@IanMadd Would it be possible to put "Chef" in front of the different "Automate" instances for consistency?